tests/mock_tables/appl_db.json

```diff
@@ -135,8 +135,17 @@
         "role": "Int",
         "speed": "100000"
     },
-    "INTF_TABLE:Ethernet0.10": {
-        "admin_status": "up"
+    "INTF_TABLE:Ethernet16": {
+        "mpls": "enable"
+    },
```

Comment:
are these changes required since you are adding tests only for multi ASIC

Reply:
reverted